WeakAuras/RegionTypes/AuraBar.lua

```diff
@@ -1188,7 +1191,8 @@ local function modify(parent, region, data)
 
   function region:SetTime(duration, expirationTime, inverse)
     local remaining = expirationTime - GetTime();
-    local progress = duration ~= 0 and remaining / duration or 0;
+    local adjustedDuration = region.useStaticDuration and region.staticDuration or duration
```

Comment:
I put this here because I think staticDuration is only really useful for bars. I also considered [WeakAuras.regionPrototype.AddSetDurationInfo()](https://github.com/WeakAuras/WeakAuras2/blob/6bc18c124767e6fb29e022ee0bb54e32f41b1284/WeakAuras/RegionTypes/RegionPrototype.lua#L106) but I'm too unfamilar with the code to be sure that the duration isn't updated somewhere between `region:SetDurationInfo()` and `region:SetTime()`